graylog2-web-interface/src/components/graylog/Accordion.test.tsx

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+import * as React from 'react';
+import { render, screen } from 'wrappedTestingLibrary';
+
+import Accordion from './Accordion';
+import AccordionItem from './AccordionItem';
+
+describe('Accordion', () => {
+  it('should render with all Items closed', () => {
+    render(
+      <Accordion id="accordion-test">
+        <AccordionItem name="Example A">
+          <h4>A as in Apple</h4>
+        </AccordionItem>
+
+        <AccordionItem name="Example 2">
+          <h4>Two as in Teeth</h4>
+        </AccordionItem>
+
+        <AccordionItem name="Example III">
+          <h4>Three as in Dimensions</h4>
+        </AccordionItem>
+      </Accordion>,
+    );
+
+    // @ts-ignore
+    const expandedButtons = screen.queryAllByRole('button', { expanded: false });
+
+    expect(expandedButtons.length).toBe(3);
+  });
+
+  it('should render with one item opened', () => {
+    render(
+      <Accordion onSelect={jest.fn()}
+                 id="accordion-test"
+                 activeKey="example-a">
+        <AccordionItem name="Example A">
+          <h4>A as in Apple</h4>
+        </AccordionItem>
+
+        <AccordionItem name="Example 2">
+          <h4>Two as in Teeth</h4>
+        </AccordionItem>
+
+        <AccordionItem name="Example III">
+          <h4>Three as in Dimensions</h4>
+        </AccordionItem>
+      </Accordion>,
+    );
+
+    // @ts-ignore
+    const expandedButtons = screen.queryAllByRole('button', { expanded: false });
+
+    expect(expandedButtons.length).toBe(2);
```

Comment:
I'd suggest we also test which of the items is opened. So in this case if "A as in Apple" is visible somewhere.

Reply:
Any idea why it's returning three items here? One of them for sure has aria-expanded=true but it's still returning with the query for expanded: false

Reply:
I've never used a query by aria role, so I am not sure. Does it reflect your expected behaviour in browser regarding the aria attributes?

Reply:
Updated dep in #9892 fixes this

Reply:
Perfect!